app/logging/LogStash.scala

```diff
@@ -62,9 +62,9 @@ object LogStash {
         lb.info("Configuring Logback")
         val context = lb.getLoggerContext
         val customFields = Map(
-          "stack" -> "fronts",
+          "stack" -> "cms-fronts",
           "stage" -> config.environment.stage.toUpperCase,
-          "app"   -> config.logging.app
+          "app"   -> config.environment.applicationName
```

Comment:
Nothing in this pr changed the `ApplicationConfiguration` but I'm going to assume that `environment.applicationName` already exists...